test/files/pse001/pass-pse001-ifcpropertyset_type_check_4x3.ifc

```diff
@@ -11,7 +11,7 @@ DATA;
 #4=IFCSIUNIT(*,.VOLUMEUNIT.,.MILLI.,.CUBIC_METRE.);
 #5=IFCSIUNIT(*,.TIMEUNIT.,$,.SECOND.);
 #6=IFCUNITASSIGNMENT((#5,#4,#2,#3));
-#7=IfcWallType('2nJrDaLQfJ1QPhdJR0o97J',$,$,$,$,(#8),$,$,$,$);
```

Comment:
Do we have a check for this or did you find this by accident?

Reply:
It failed the syntax check
![image](https://github.com/user-attachments/assets/ef335eb2-7cab-4277-82a6-f66292a7680e)

Reply:
Thanks :)